packages/esbuild-why/test/index.test.js

```diff
@@ -1,12 +1,15 @@
-let {readFile} = require('fs').promises
-let [esbuild] = require('@size-limit/esbuild')
-let {join} = require('path')
-let rm = require('size-limit/rm')
-let open = require('open')
+import esbuildPkg from '@size-limit/esbuild'
+import {readFile} from 'fs/promises'
+import open from 'open'
+import {join} from 'path'
+import rm from 'size-limit/rm'
+import { afterEach, expect, it, vi } from "vitest"
```

Comment:
Can you run Prettier on changed files?

I have it in VS Code. But you can install to the project, run, and uninstall.

Reply:
Ah, damn, I did `eslint --fix` but forgot about prettier. I've installed it globally and ran `pnpm exec prettier **/*.js --write`, should do the trick.